apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -259,6 +266,43 @@
          * @see org.apache.skywalking.apm.agent.core.logging.core.PatternLogger#DEFAULT_CONVERTER_MAP
          */
         public static String PATTERN = "%level %timestamp %thread %class : %msg %throwable";
+
+        public static class JSON {
+            /**
+             * Key of the AgentName
+             */
+            public static String AGENT_NAME_KEY = "agent_name";
+
+            /**
+             * Key of the timestamp
+             */
+            public static String TIMESTAMP_KEY = "@timestamp";
```

Comment:
Why starting with `@` in JSON?

Reply:
Just a convention used in [`logstash-logback-encoder`](https://github.com/logstash/logstash-logback-encoder#providers-for-loggingevents)

Reply:
Why we need to adopt logstash?

Reply:
> Why we need to adopt logstash?

I suppose it would be better to comply to the conventions which are common used by others?

Reply:
Discussion continues on https://github.com/apache/skywalking/pull/5357/files#r473956466